src/components/common/AppBars/CategoryBar.js

```diff
@@ -24,13 +24,18 @@ class Categories extends Component {
                 }}
                 key={Number(index)}
                 className={category === focused ? "current" : null}
+                data-test="single-category"
               >
                 {category}
               </li>
             ))}
           </div>
         </div>
-        <div onClick={() => onMoreClick()} className="category-right">
+        <div
+          onClick={() => onMoreClick()}
```

Comment:
What do you think about passing your click handler as a reference instead?

Reply:
Personally, I don't prefer references if there is a way to declary function and use it directly. 

<img width="944" alt="Screen Shot 2019-05-07 at 10 58 21" src="https://user-images.githubusercontent.com/20681465/57287162-30b4eb80-70b7-11e9-87fb-3590d5dd6ed9.png">